app/editor/menus/attachment.tsx

```diff
@@ -1,13 +1,80 @@
-import { TrashIcon, DownloadIcon, ReplaceIcon } from "outline-icons";
+import {
+  TrashIcon,
+  DownloadIcon,
+  ReplaceIcon,
+  AlignImageLeftIcon,
+  AlignImageRightIcon,
+  AlignImageCenterIcon,
+  AlignFullWidthIcon,
+} from "outline-icons";
 import { EditorState } from "prosemirror-state";
 import { MenuItem } from "@shared/editor/types";
 import { Dictionary } from "~/hooks/useDictionary";
+import { isNodeActive } from "@shared/editor/queries/isNodeActive";
 
 export default function attachmentMenuItems(
   state: EditorState,
   dictionary: Dictionary
 ): MenuItem[] {
+  const { schema } = state;
+  const isLeftAligned = isNodeActive(schema.nodes.attachment, {
+    layoutClass: "left-50",
+  });
+  const isRightAligned = isNodeActive(schema.nodes.attachment, {
+    layoutClass: "right-50",
+  });
+  const isFullWidthAligned = isNodeActive(schema.nodes.attachment, {
+    layoutClass: "full-width",
+  });
+  const isPreview = isNodeActive(schema.nodes.attachment, {
+    preview: true,
+  });
+
   return [
+    {
```

Comment:
I would remove all these alignment options in a first version, resizing is enough – also means we don't need to bring across the legacy `layoutClass` stuff from images.